prql-js/README.md

```diff
@@ -1,61 +1,66 @@
 # prql-js
 
-JavaScript bindings for [`prql-compiler`](https://github.com/prql/prql/). Check
-out <https://prql-lang.org> for more context.
-
-This uses
-[`wasm-pack`](https://rustwasm.github.io/docs/wasm-pack/tutorials/npm-browser-packages/index.html)
-to generate bindings[^1].
-
-[^1]:
-    though we would be very open to other approaches, and used `trunk`
-    successfully in a rust-driven approach to this, RIP `prql-web`.
+JavaScript bindings for [`prql-compiler`](https://github.com/prql/prql/). Check out <https://prql-lang.org> for more
+context.
 
 ## Installation
 
-To install the currently published version:
-
 ```sh
 npm install prql-js
 ```
 
-This package is built to target a bundler (i.e. webpack). To use it with Node.js
-or import it directly in a browser as an ES module, [build it using a suitable
-`--target`](https://rustwasm.github.io/docs/wasm-pack/commands/build.html).
-
 ## Usage
 
-```js
-import compile from "prql-js";
+Currently the `compile(prql_string)` function is the only function available.
 
-const { sql, error } = compile(`from employees | select first_name`);
+### From NodeJS
+
+```javascript
+const prql = require("prql-js/dist/node/prql_js.js");
+
+const sql = prql.compile("from employees | select first_name").sql;
```

Comment:
I had a merge conflict here — is this correct? `compile` returns an `Option` in rust

Reply:
I would use their example for compile the destructuring of SQL and error looks better